rosdep/base.yaml

```diff
@@ -4153,14 +4153,32 @@ wxpython:
     xenial: [python-wxgtk3.0]
 wxwidgets:
   arch: [wxgtk]
-  debian: [libwxgtk2.8-dev]
+  debian:
+    jessie: [libwxgtk3.0-dev]
+    stretch: [libwxgtk3.0-dev]
```

Comment:
Thanks for the debian updates. I removed the sid entry since we don't target sid because it's a rolling target.